dnscache.py

```diff
@@ -163,22 +162,18 @@ def get(self, host, port=0):
                     continue
 
                 ip = records[-1][0]
-                if not ip:
-                    del self.lookup_table[lookup_name]
-                    raise Exception(
-                        "No IP records found in record set %s" % records)
+
+                self.mark_hostport_down(host, ip, port)
```

Comment:
This will fail with 

```
>>> d.get(host='google.com', port=443)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "[...]/tecart-http-proxy/dnscache.py", line 168, in get
    if ip not in self.lookup_table[lookup_name]:

```

Reply:
Thank you for your contribution. I like the PEP8 fixes, but there's a bug that breaks `DnsCache.get()`